### PR TITLE
whitelist pulp

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
whitelist pulp

### Linked issues
Progresses #1901 

### Functionality 
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -21430,6 +21430,40 @@
...
+    "pulp.pulp": [],
+    "pulp.solverdir": [],
+    "pulp.solverdir.cbc.linux.32": [],
+    "pulp.solverdir.cbc.linux.64": [],
+    "pulp.solverdir.cbc.linux.arm64": [],
+    "pulp.solverdir.cbc.osx.64": [],
+    "pulp.solverdir.cbc.win.32": [],
+    "pulp.solverdir.cbc.win.64": [],
+    "pulp.sparse": [],
+    "pulp.tests": [],
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:21459]: We remove the tests, right?

File: src/databricks/labs/ucx/source_code/known.json [hunk 2]
```diff
@@ -21430,6 +21430,40 @@
     "ptyprocess.ptyprocess": [],
     "ptyprocess.util": []
   },
+  "pulp": {
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:21433]: Same as for #2071, looks like a library that does not spark

**Review by JCZuurmond — COMMENTED** [review]
See comments

File: src/databricks/labs/ucx/source_code/known.json (hunk 2 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:21433]: It's in scope with solacc, so there must be use case scenario.

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:21459]: done

**Review by JCZuurmond — APPROVED** [review]

**Pull request merged**